src/mail/mjml/warning_email.mjml

```diff
@@ -0,0 +1,44 @@
+<mjml>
+  <mj-include path="./partial/header.mjml" />
+  <mj-body mj-class="bg--blue-100">
+    <mj-wrapper css-class="wrapper" padding="20px 40px 40px 40px">
+    <mj-section>
+      <mj-column>
+        <mj-image src="{% base64_static 'images/logo_fun.png' %}" width="200px" align="left" alt="Logo de France Université Numérique" />
```

Comment:
Is it working ? I don't see anywhere a `{% load %}` tag

Reply:
[This test](https://github.com/openfun/mork/pull/14/files#diff-7b95d586f42af7c96122801841f113947fc3f1597d27b41531f9ac61f6b67d80R29) makes sure it is working!
With pure Jinja2, it seems that we don't load custom tags from inside the template, but at the `Environment` instantiation (see [here](https://github.com/openfun/mork/pull/14/files#diff-43ab2c84c38e6cc27675d454bb7c8e5cd355ce79061eb2638863e78b4f2e3492R28).

Reply:
Ok right !